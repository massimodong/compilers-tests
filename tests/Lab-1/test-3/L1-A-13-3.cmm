MbJJzG ( struct {
}J , struct uyAuPdNrk3u2nk7U7XD {
}U3j_Gcw56t ) {
  struct CzsoxyrA4GHOu FWXEN3JA . ;
  float bo2pZU1JvQnN ;
  if ( bQpacafIUi5l3BXw )   J02nFLR8Wp5UOdoc ;
else   if ( xbQI ) {
    int Wqrrh [ 1613406591 ] , jRHD0Y = XDYPT ;
    while ( M )     vkxbILgU ;

    if ( L8AfZ92MytI4dmdP     while ( THsx7gd70DqnO 
      )     if LvFENH9YEHZcSR )     tt8BFjywukTN ;
else     Eq2jTPuuhb0wPdSYXt ;


else     if ( PYHDVVsxcnyiWmBy )     return qOdIAQ0J2HC1II5Z 
      || msHfXXxG9RiutfxR1dHB >= n / ( _AOY5VYVgAa7a = xDLh 
      && PtaKMdkDg28xBA > jebuo3 + t78ncH1K7Muf3t [ Bsa_OIYbRQ 
      <= I85m7m0OloAlTpj - EBLgQkm ( j , fzE7vAgQEnXm && 
      eOmscUsdvMsT9CYt7Qm + e27FeIE0gEIbYUtBk * J7RO0te2q 
      ( CRP_J1HTn4CxX || K < O2GCxWTR ( = rN29Y [ sztYNPmonXV 
      && y2OlFEbVE1_d2TaPkf ( iZwx5w + - 1523178481.1159165629 
      ) . NWuJKOVzc_YVOAtcI2b ] == ge ) ) != - 947146738 
      ) ] ) . NWhedxo31AR * ! - lrd0T68Iv ( ) ;


@  }

}
int ;
int iXpuwAqHDU80p3hjm , BYhGlfkX6S4JB0 
#
;
int mh ( ) {
}
int ;`
int ;
