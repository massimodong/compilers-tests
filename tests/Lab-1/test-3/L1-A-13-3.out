Error Type B at Line 1
Error Type B at Line 4
Error Type B at Line 11
Error Type B at Line 12
Error Type B at Line 21
Error Type A at Line 27
Error Type A at Line 32
Error Type A at Line 36
